tokio/src/runtime/metrics/runtime.rs

```diff
@@ -578,6 +579,50 @@ impl RuntimeMetrics {
         self.handle.inner.worker_local_queue_depth(worker)
     }
 
+    pub fn poll_count_histogram_enabled(&self) -> bool {
```

Comment:
Following from my other comment, if we expose a (non-exhaustive) `PollCountHistogramConfig` struct, then these 4 methods could be replaced by one that returns `Option<PollCountHistogramConfig>`.